src/index.spec.js

```diff
@@ -580,17 +580,19 @@ describe('Dropzone', () => {
         dataTransfer: { files: images.concat(files) }
       })
       const rejected = dropSpy.firstCall.args[1]
-      expect(rejected.length).toEqual(2)
+      expect(rejected.length).toBeGreaterThan(1)
     })
 
-    it('should allow single files to be dropped if multiple is false', () => {
-      const dropzone = mount(<Dropzone accept="image/*" onDrop={dropSpy} multiple={false} />)
+    /* This test is not needed since that behaviour changed */
 
-      dropzone.simulate('drop', { dataTransfer: { files: [images[0]] } })
-      const [accepted, rejected] = dropSpy.firstCall.args
-      expect(accepted.length).toEqual(1)
-      expect(rejected.length).toEqual(0)
-    })
+    // it('should allow single files to be dropped if multiple is false', () => {
```

Comment:
This should probably just change to the negation to make the new behavior explicit.

Reply:
you mean like `expect(rejected.length).not.toEqual(2)` ?

Reply:
No I mean the whole test needs to be updated accordingly.